api/src/main/resources/dev.db.properties

```diff
@@ -0,0 +1,6 @@
+application.gifts.datasource.driverClassName=org.postgresql.Driver
+application.gifts.datasource.jdbcUrl=jdbc:postgresql://localhost:5432/gift_db
```

Comment:
looks like these properties are redundant and should be replaced with `spring.datasource.` ones